modules/systemd/default.nix

```diff
@@ -40,7 +45,7 @@ with lib; {
 
         # Link the X11 socket into place. This is a no-op on a normal setup,
         # but helps if /tmp is a tmpfs or mounted from some other location.
-        tmpfiles.rules = [ "L /tmp/.X11-unix - - - - ${cfg.wslConf.automount.root}/wslg/.X11-unix" ];
+        tmpfiles.rules = mkIf cfg.wslgMountX11 ([ "L /tmp/.X11-unix - - - - ${cfg.wslConf.automount.root}/wslg/.X11-unix" ]);
```

Comment:
The () here are not required if I eyeballed that correct.

Also can't we use the new tmpfiles syntax to easily disable this with mkForce? I would prefer that over a new setting.

Reply:
Agreed, using the new tmpfiles syntax from now on sounds like a good Idea. 
There should be a single setting to disable everything related to WSLg (see #266). If there's a need to disable the tmpfiles rule on its own, it could still be overriden with mkForce as @SuperSandro2000 said

Reply:
Thanks for the Feedback, i didn't know about the new tmpfiles syntax.

I updated the PR and tested it, overriding works fine so this is a better solution.